snippets/ingress.yaml

```diff
@@ -1,8 +1,4 @@
-name: ingress
-label: Kubernetes Ingress
-description: Kubernetes Ingress
-body: |2 
```

Comment:
Can you add these back? It looks like you not only changed the apiVersion but you also changed the snippet.

Reply:
Oops! Sorry about that